pydoctor/driver.py

```diff
@@ -96,6 +93,14 @@ def get_supported_docformats() -> Iterator[str]:
         else:
             yield moduleName
 
+def get_themes() -> Iterator[str]:
```

Comment:
I don't want to add code to this package because to me it's only a data package.

I've added the `__init__.py` file because I had to, to use importlib.

Reply:
Maybe it started as a data package, but if there is theme-specific code like this, doesn't it make sense to put that under `pydoctor.themes`?

Reply:
Well, that's not my opinion but it's not a big deal. I'll move the code there